### PR TITLE
About us

### PR DESCRIPTION
**Pull request by mg27713**
Merge from branch `mg27713`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,7 +63,19 @@ <h1 class="letter">N</h1>
...
+  
+  <div class="container">
+    <h2>About us</h2>
+    <p>
+        We are <a class="link" href="https://gunn.pausd.org">Gunn</a> people who often sit in the N building of
+      <a class="link" href="https://tinyurl.com/GunnDiscord" target="_blank">Gunn Discord</a>.
+    </p>
+    
+    <p>
+      We do <b>NOT</b> live under the stairs or in the basement of the N building. Please recheck your sources.
```
Comment (SheepTester) [inline index.html:77]: Good.

**Review by SheepTester — APPROVED** [review]

**Comment by SheepTester** [comment]
By the way, if you do `target="_blank"`, you should include `rel="noopener noreferrer"`. Otherwise, the opened site has control over the page it was opened from (it can redirect) the page. However, I trust tinyurl.com so I think it's fine

**Comment by mg27713** [comment]
Interesting. I didn't know `<a>` had a `rel` attr. `noreferrer` is definitely a good idea though and I think browsers should have an option to do `noreferrer` by default, and websites should be able to explicitly say `referrer` if they need one.

**Comment by SheepTester** [comment]
> Interesting. I didn't know `<a>` had a `rel` attr. `noreferrer` is definitely a good idea though and I think browsers should have an option to do `noreferrer` by default, and websites should be able to explicitly say `referrer` if they need one.

Well, yes, but that'd break old websites relying on this behaviour. Can't have that happen!

**Pull request merged**